lib/fizzy/parser_expr.cpp

```diff
@@ -148,6 +150,12 @@ parser_result<Code> parse_expr(
     // The function's implicit block.
     control_stack.emplace(Instr::block, function_arity, 0);
 
+    // This is needed for aiding validation.
+    assert((uint64_t{local_count} + module.typesec[func_type_idx].inputs.size()) <=
+           std::numeric_limits<uint32_t>::max());
+    const uint32_t max_local_index =
```

Comment:
Well it's not max index, maybe `total_local_arg_count` or `local_and_arg_count`

Reply:
Why isn't it max index? The immediate it is compared against is called "local index".

Reply:
Because it's max index + 1